circle.yml

```diff
@@ -379,6 +379,9 @@ jobs:
             prague/eip2537_bls_12_381_precompiles/bls12_g2mul
             prague/eip2537_bls_12_381_precompiles/bls12_g1msm
             prague/eip2537_bls_12_381_precompiles/bls12_g2msm
+            prague/eip2537_bls_12_381_precompiles/bls12_map_fp_to_g1
+            prague/eip2537_bls_12_381_precompiles/bls12_map_fp2_to_g2
+            prague/eip2537_bls_12_381_precompiles/bls12_pairing
```

Comment:
Likely we should be able to execute all tests in the `eip2537_bls_12_381_precompiles` directory.